shared/constants/types/more.tsx

```diff
@@ -1,56 +1,64 @@
-const provablePlatformsMap = {
-  dns: true,
-  facebook: true,
-  github: true,
-  hackernews: true,
-  http: true,
-  https: true,
-  reddit: true,
-  rooter: __DEV__,
-  twitter: true,
-  web: true,
-}
+export type ProvablePlatformsType =
+  | 'dns'
+  | 'facebook'
+  | 'github'
+  | 'hackernews'
+  | 'http'
+  | 'https'
+  | 'reddit'
+  | 'rooter'
+  | 'twitter'
+  | 'web'
+  | 'rooter'
```

Comment:
Any reason why we didn't go with enum here? Not suggesting a change necessarily; just curious